ui/app/styles/core/helpers.scss

```diff
@@ -213,6 +213,9 @@
 .has-top-margin-xxl {
   margin-top: $spacing-xxl;
 }
+.has-top-margin-negative-s {
```

Comment:
For the new component, needed them to be closer to the label so they seemed "grouped" together.